.github/workflows/test.yml

```diff
@@ -15,12 +15,16 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest]
         python-version: ["3.8", "3.9", "3.10", "3.11"]
-        pandas-version: ["1.3.5", "1.4.4", "1.5.3", "2.2.2"]
+        pandas-version: ["1.3.5", "1.4.4", "1.5.3", "2.0.3", "2.1.4", "2.2.2"]
```

Comment:
I think testing over two versions would be enough, i.e., 1.5.3 and 2.2.2. Is there any specific reason to test for 1.3.5, 1.4.4, 2.0.3, and 2.1.4?

Reply:
No specific reason. I'm just following current convention (1.3, 1.4, 1.5)

If you think it's uneccessary, I'll test major versions only. I think testing minor version has limited benefit as well (unless there is some bug in one of pandas versions). Testing only 2 versions will drastically improve test time (now close to 1 hour 😬 )

Reply:
Okay, then let's test 2 major versions only.

Reply:
Updated in https://github.com/treasure-data/pytd/pull/138/commits/cad4317ed2e953b3b0c5dc769a43a268300d9acd